CMakeLists.txt

```diff
@@ -1,12 +1,16 @@
 #TODO: write FindxxHash for libxxhash-dev compatibility?
 #Cmake for xxHash is unofficial, so libxxhash-dev doesn't have xxHashConfig.cmake
 
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
why?

Reply:
Needed for openmp
3.1 doesn't support the cmake commands used

Reply:
alright